### PR TITLE
feat: change name to SUPPORT

### LINKED ISSUE #42: Change name from DOCUMENT to SUPPORT
## Problem

The common industry practice is to name user documentation something along the lines of support or help centre. The current name of document does not communicate the user documentation in a manner all users would easily understand and then navigate to to help their own platform use.

## Solution

Change the user documentation from DOCUMENT to SUPPORT. This needs to be applied within the user documentation code itself as well as the url links, i.e. support.gliff.app to CNAME etc.

The updated links will then need to be applied within the platform itself.

## Considered Alternatives

N/A

### PR DESCRIPTION
**Pull request by ChasNelson1990**
## Description

Changes all references to "DOCUMENT" to "SUPPORT" including changing this repos name and description.

Docs can now be accessed at either:

https://docs.gliff.app
https://support.gliff.app

closes #42

### CODE REVIEW

**Review by joshuajames-smith — COMMENTED** [review]
Screenshot 2022-05-10 at 13 42 23

This appears but only on support.gliff.app? Not sure if it's anything that would resolve itself?

**Review by joshuajames-smith — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-DOCUMENT is [gliff.ai](https://gliff.ai)'s user documentation for supporting use of our user-friendly interface for curating, annotation and collaborating with imaging datasets for the purposes of developing imaging AI products. When the full [gliff.ai platform](https://gliff.ai/software/) is used, DOCUMENT provides a clear guide for our users to support the development of high-quality and auditable datasets that satisfy any relevant regulatory frameworks which enables our users to build world-changing and trustworthy AI models and products.
+SUPPORT is [gliff.ai](https://gliff.ai)'s user supportation for supporting use of our user-friendly interface for curating, annotation and collaborating with imaging datasets for the purposes of developing imaging AI products. When the full [gliff.ai platform](https://gliff.ai/software/) is used, SUPPORT provides a clear guide for our users to support the development of high-quality and auditable datasets that satisfy any relevant regulatory frameworks which enables our users to build world-changing and trustworthy AI models and products.
```
Comment (RoseBunker) [inline README.md]: Pretty sure 'supportation' doesn't work and the regulatory bit is too strong, so what about stripping it back to:
"SUPPORT is gliff.ai's guide for users who are curating, annotating and collaborating to build high-quality, auditable datasets for imaging AI."

**Review by RoseBunker — CHANGES_REQUESTED** [review]
Just one comment left, suggested wording change. We can't say our product will satisfy any regulatory requirements - 'satisfy' is too strong a word, we help; and 'any' is also too big; we can't guarantee these things.

File: README.md (hunk 1 above)
Comment (ChasNelson1990) [reply README.md]: Ahahah! I think supportation has come from a find all and replace action. We should check there aren't any more!

**Review by joshuajames-smith — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
-DOCUMENT is [gliff.ai](https://gliff.ai)'s user documentation for supporting use of our user-friendly interface for curating, annotation and collaborating with imaging datasets for the purposes of developing imaging AI products. When the full [gliff.ai platform](https://gliff.ai/software/) is used, DOCUMENT provides a clear guide for our users to support the development of high-quality and auditable datasets that satisfy any relevant regulatory frameworks which enables our users to build world-changing and trustworthy AI models and products.
+SUPPORT is [gliff.ai](https://gliff.ai)'s user documentation for supporting use of our user-friendly interface for curating, annotation and collaborating with imaging datasets for the purposes of developing imaging AI products. When the full [gliff.ai platform](https://gliff.ai/software/) is used, SUPPORT provides a clear guide for our users to support the development of high-quality and auditable datasets that satisfy any relevant regulatory frameworks which enables our users to build world-changing and trustworthy AI models and products.
```
Comment (RoseBunker) [inline README.md:9]: I'm sorry for being a pain but the wording "satisfy any relevant regulatory frameworks" is still too problematic for me. The whole sentence sounds far to much like we can guarantee that their AI will be regulatable. You could just remove the whole sentence - it's not necessary to have this type of messaging in the SUPPORT intro anyway.

File: README.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ Looking for something specific? 🔍
 
 [{{back to navigation}}](#table-of-contents)
 
-Initially you will need to install [Ruby <2.7](https://www.ruby-lang.org/en/documentation/installation/) and [bundler](https://bundler.io/) with `gem install bundler`.
+Initially you will need to install [Ruby <2.7](https://www.ruby-lang.org/en/supportation/installation/) and [bundler](https://bundler.io/) with `gem install bundler`.
```
Comment (RoseBunker) [inline README.md:33]: I accept that "supportation" is a word, but it's a terrible word, I don't think we should use it - it sounds made up. Can we just say support or SUPPORT ?

**Review by RoseBunker — CHANGES_REQUESTED** [review]
Sorry if submitted Review twice, have GitHub message telling me to put a comment here... please see my comment on the "satisfy any..." wording. I think cut that whole sentence out.

File: README.md (hunk 2 above)
Comment (ChasNelson1990) [reply README.md:9]: Is this not dealt with in @philhallbio 's PR?

Comment (ChasNelson1990) [reply README.md:9]: @philhallbio ?

Comment (philhallbio) [reply README.md:9]: It isn't currently but happy to add it in to those changes

**Pull request merged**